Correct clock default time (am/pm) when value exists

donluz opened the pull request:
Hoping to get the issue of re-selecting the control to reveal the clockface when a value already exists in the control.  The time is correct, but the "am/pm" is defaulting to "am" and not examining/reflecting the already-selected time.

Thanks for any efforts & attention you can give to this.

JordyMoos committed:
Allow to multi increment minutes and hours

JordyMoos committed:
Added the new options to the readme

JordyMoos committed:
Fix tab/spaces issue

JordyMoos committed:
Fix tab/spaces issue

JordyMoos committed:
Fix tab/spaces issue

JordyMoos committed:
Fix tab/spaces issue

JordyMoos committed:
Bug fix with am/pm

JordyMoos committed:
Fix tab/spacing

JordyMoos committed:
- Added the getTime function
- Allow to add a Date as the 'default' options

JordyMoos committed:
Wrong lookup to the twelvehour option

JordyMoos committed:
Add intellij idea to gitignore

JordyMoos committed:
Merge remote-tracking branch 'origin/get-datetime'

Conflicts:
	README.md
	src/clockpicker.js

JordyMoos committed:
Merged the get-datetime branch.
Also updated the minified versions

JordyMoos committed:
Updated the readme for the version

technopahadi committed:
adding a new option to allow submit on clicking on am/pm rather than clicking the minute pickers/options

JordyMoos committed:
Merge pull request #2 from rishabhp/master

adding a new option to allow submit on clicking on am/pm rather than clicking the minute pickers/options

JordyMoos commented:
This PR should be pointed to https://github.com/JordyMoos/clockpicker ;)

Edit:
It does not have any of your commits in it?

JordyMoos committed:
Fix where time swapped back to am

JordyMoos committed:
Merge pull request #6 from JordyMoos/am-pm

Fix where time swapped back to am

Commit:
Added the minified.
Also added @rishabhp submit feature to the source so it shows up in all dist files

JordyMoos committed:
Merge pull request #7 from JordyMoos/am-pm

Added the minified.

flibdev committed:
Added support for HTML5 input fields

flibdev committed:
Bug fix for 12 and 00 hours being swapped

flibdev committed:
Enabled & tested HTML5 time output

flibdev committed:
Fixed logic for HTML5 output

flibdev committed:
Readded magically missing lines

flibdev committed:
Updated README with new options

JordyMoos committed:
Merge pull request #9 from flibX0r/master

Enabling support for HTML5 input type="time"

madand committed:
Implement adaptive vertical placement

JordyMoos committed:
Merge pull request #10 from MadAnd/adaptive-vertical-placement

Implemented adaptive vertical placement

madand committed:
Build distfiles with the adaptive placement patch

JordyMoos committed:
Merge pull request #11 from MadAnd/build-adaptive-js

Build distfiles with the adaptive placement patch

ErDmKo committed:
Triggers and callbacks try

ErDmKo committed:
Insert time to callbacks

ErDmKo committed:
second time try

ErDmKo committed:
Trigger ankother try

JordyMoos committed:
Merge pull request #12 from ErDmKo/master

Extension for callbacks and triggers

Commit:
triggerhandler to trigger event

JordyMoos committed:
Merge pull request #16 from srinivasdamam/master

triggerhandler to trigger event
